# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libpcap/pcap-config-nospace.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- libpcap-1.0.0/pcap-config.in.orig	2010-04-22 11:23:34.724075461 +0200
+++ libpcap-1.0.0/pcap-config.in	2010-04-22 11:24:15.664339594 +0200
@@ -7,10 +7,10 @@
 case "$1" in
 
 --cflags)
-	echo "-I @includedir@"
+	echo "-I@includedir@"
 	;;
 
 --libs)
-	echo "-L @libdir@ -lpcap @DEPLIBS@"
+	echo "-L@libdir@ -lpcap @DEPLIBS@"
 	;;
 esac
